deployment/README.md

```diff
@@ -191,30 +191,43 @@ Databricks allows you to submit Spark .NET apps to an existing active cluster or
 > **Note:** This approach allows job submission to an existing active cluster.
 
 One-time Setup:
-  1. Go to your Databricks cluster -> Jobs (on the left-side menu) -> Set JAR
-  2. Upload the appropriate `microsoft-spark-<spark-version>-<spark-dotnet-version>.jar`
-  3. Set the params appropriately:
-     ```
-     Main Class: org.apache.spark.deploy.DotnetRunner
-     Arguments /dbfs/app/<your-app-name>.zip <your-app-main-class>
-     ```
-  4. Configure the Cluster to point to an existing cluster (that you already set the init script for - see previous section).
-  
+   1. Go to your Databricks cluster -> Jobs (on the left-side menu) -> Set JAR
+   2. Upload the appropriate `microsoft-spark-<spark-version>-<spark-dotnet-version>.jar`
+   3. Set the params appropriately:
+      ```
+      Main Class: org.apache.spark.deploy.DotnetRunner
+      Arguments /dbfs/apps/<your-app-name>.zip <your-app-main-class>
+      ```
+   4. Configure the Cluster to point to an existing cluster (that you already set the init script for - see previous section).
+
 Publishing your App & Running:
-  1. You should first [publish your app](#preparing-your-spark-net-app). 
-  2. Use [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html) to upload your application to Databricks cluster. For instance, 
-     ```
-     cd <path-to-your-app-publish-directory>
-     databricks fs cp <your-app-name>.zip dbfs:/apps/<your-app-name>.zip
-     ```
-  3. Now, go to your Databricks cluster -> Jobs -> <Job-name> -> Run Now to run your job!
+   1. You should first [publish your app](#preparing-your-spark-net-app). 
+      > **Note:** Do not use `SparkSession.Stop()` in your application code when submitting jobs to an existing active cluster.
+   2. Use [Databricks CLI](https://docs.databricks.com/user-guide/dev-tools/databricks-cli.html) to upload your application to Databricks cluster. For instance, 
+      ```shell
+      cd <path-to-your-app-publish-directory>
+      databricks fs cp <your-app-name>.zip dbfs:/apps/<your-app-name>.zip
+      ```
+   3. This step is only required if app assemblies (e.g., DLLs that contain your user-defined functions, libraries that the UDFs depend on) need to be placed in the working directory of each Microsoft.Spark.Worker.
+      - Upload your application assemblies to your Databricks cluster
+         ```shell
+         cd <path-to-your-app-publish-directory>
+         databricks fs cp <assembly>.dll dbfs:/apps/dependencies
+         ```
+      - Further modify **[db-init.sh](../deployment/db-init.sh)** appropriately to point to your app dependencies and upload to your Databricks cluster
```

Comment:
I would add more details here (and even an example). It is not clear what we mean when we say further modify.

Reply:
Looks like you already have an example in the script? Maybe forward reference and indicate there's an example in the script?